docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/DetectIndependentIdenticallyDistributedChangePoint.cs

```diff
@@ -111,7 +111,7 @@ public static void IidChangePointDetectorPrediction()
             string inputColumnName = nameof(IidChangePointData.Value);
 
             // Time Series model.
-            ITransformer model = ml.Transforms.IidChangePointEstimator(outputColumnName, inputColumnName, 95, Size / 4).Fit(dataView);
+            ITransformer model = ml.Transforms.DetectIndependentIdenticallyDistributedChangePoint(outputColumnName, inputColumnName, 95, Size / 4).Fit(dataView);
```

Comment:
@CESARDELATORRE, @eerhardt, @TomFinley @shauheen  how do you feel about those API names?

I would rather leave it to `DetectIIDChangePoint`, but abbreviations have been deemed cryptic.

Reply:
I would stick with IID, it is fairly well known abreviation.

Reply:
We have Lda, Pca, and so on. I feel `Iid` is not much worse than them.

Reply:
thanks for the input. 

---
In reply to: [265343959](https://github.com/dotnet/machinelearning/pull/2900#discussion_r265343959) [](ancestors = 265343959)